taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "license_number", "first_name", "last_name", "email"
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta(forms.ModelForm):

```

Comment:
The `Meta` class in `DriverLicenseUpdateForm` should inherit from `forms.ModelForm.Meta` instead of `forms.ModelForm`. This ensures that the form's metadata is correctly set up.